app/crud/storage.py

```diff
@@ -0,0 +1,42 @@
+from typing import Generator, Iterator, BinaryIO
+
+from minio import S3Error
+from minio.datatypes import Object
+from minio.helpers import ObjectWriteResult
+
+from app.config import minio_settings
+from app.storage import storage
+
+
+def is_file_exists(path_to_object: str) -> bool:
+    try:
+        if storage.stat_object(minio_settings.BUCKET_NAME, path_to_object) != None:
+            return True
+        else:
+            return False
```

Comment:
А сам говорил, что не любишь вложенность. Можно написать в одну строку.

```suggestion
        return storage.stat_object(minio_settings.BUCKET_NAME, path_to_object) is not None
```

Reply:
Да тут вложенность небольшая, но справедливо